unit/api/customer_resource.py

```diff
@@ -4,8 +4,8 @@
 
 
 class CustomerResource(BaseResource):
-    def __init__(self, api_url, token):
-        super().__init__(api_url, token)
+    def __init__(self, api_url, token, retries):
+        super().__init__(api_url, token, retries)
```

Comment:
maybe retries can be set with default values so in cases like here where we are not using retries since there's no creation we wouldn't have to pass it..? let me know what you think:)